index.js

```diff
@@ -100,23 +128,31 @@ function myMap(array, callback) {
 // ***** Callbacks - Question 1 *****
 
 // *** Uncomment the lines below to test
-// console.log("%cCallbacks - Question 1", "color: red")
+console.log("%cCallbacks - Question 1", "color: red")
+ 
+function triple(value) {
+  console.log(value * 3)
+}
 
-// console.log(myMap([1,2,3,4,5], triple)) 
-// // => [3,6,9,12,15]
+console.log(myMap([1,2,3,4,5], triple)) 
+// => [3,6,9,12,15]
 
-// console.log(myMap([2,4,6,8], triple))   
-// // => [6,12,18,24]
-// console.log("%c----------", "color: red") 
+console.log(myMap([2,4,6,8], triple))   
+// => [6,12,18,24]
+console.log("%c----------", "color: red") 
 
 
 // ***** Callbacks - Question 2 *****
 // *** Uncomment the lines below to test
-// console.log("%cCallbacks - Question 2", "color: red")
+console.log("%cCallbacks - Question 2", "color: red")
 
-// console.log(myMap(["Raffy", "Chase"], greet)) 
-// // => ["Hi, Raffy!", "Hi, Chase!"]
-// console.log("%c----------", "color: red") 
+function greet(name1, name2) {
+  console.log(`"Hi, ${name1}, "Hi, ${name2}!"`)
+}
```

Comment:
Close! This one is going to be like your `triple` function. It only takes one argument at a time.

```javascript
function greet(name) {
    return `Hi, ${name}`
}
```